tests/plugins/test_plotly_extensions.py

```diff
@@ -15,6 +16,7 @@ def figure():
 def test_plotly_static_writer(figure: go.Figure, tmp_path: pathlib.Path) -> None:
     file_path = tmp_path / "figure.png"
 
+    os.environ["BROWSER_PATH"] = "/path/to/browser"  # required for PlotlyStaticWriter to work
```

Comment:
Mind adding to comments as to why this is required?

Reply:
this will be removed.